tools/sync-DotSettings/sync-DotSettings.ps1

```diff
@@ -0,0 +1,18 @@
+#Requires -Version 7.4
+
+# This script copies the Resharper/Rider code style settings (.sln.DotSettings file) from Steeltoe, 
+# patches it for use in samples, then distributes it across the samples. It enables IDE IntelliSense and code cleanup.
+# The patch file was created with: `git diff --unified=0 > DotSettings.patch`
+
+# https://stackoverflow.com/a/48877892
+$ErrorActionPreference = "Stop"
+$PSNativeCommandUseErrorActionPreference = $true
+
+Copy-Item "../../../Steeltoe/src/Steeltoe.All.sln.DotSettings" -Destination "baseline.DotSettings"
```

Comment:
Would it make more sense for this script to download the `.DotSettings` file from GitHub?

Reply:
Discussed over zoom, this script is fine as-is